extensions/wikia/RobotsTxt/classes/WikiaRobots.class.php

```diff
@@ -136,10 +135,6 @@ public function __construct( PathBuilder $pathBuilder ) {
 			}
 		}
 
-		if ( !empty( $wgAllowSpecialImagesInRobots ) ) {
-			$this->allowedSpecialPages['Images'] = 'allow';
-		}
-
 		// TODO: reverse the logic
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)